src/dom-shim.js

```diff
@@ -1,3 +1,57 @@
+/* eslint-disable no-warning-comments */
+
+import { parseFragment, serialize } from 'parse5';
+
+// TODO Should go into utils file?
```

Comment:
re: utils file, if its not needed in other files, it's probably fine to leave.  I think there are cases for a util file between wcc and jsx-loader I think, but I think for now we can probably just move the `TODO` comment, but certainly open to a PR that proposes some code structuring / organization.